### DISCUSSION TITLE
Static files issue

### DISCUSSION THREAD

**Discussion by BilousS** [netbox-community/netbox#9322] [Q&A]
### NetBox version

3.2

### Python version

3.9

### Steps to Reproduce

1. Get latest Netbox version
2. Install requirements from the attached file
3. Quick set up postgres redis (just to be running localy)
5. Fix and run migrations (as there is a migration issue with most migration files that were squashed)
6. Run manage.py collectstatic command
7. Run the server 
8. Server can be  started, nevertheless when you go to the homepage ( for example the homepage as this issue shares other endpoint that share custom templates )

### Expected Behavior
[requirements.txt](https://github.com/netbox-community/netbox/files/8651839/requirements.txt)

Expect static files to be loaded when I run the server.

### Observed Behavior

runed server locally, completed manage.py collectstatic command which created static folder in root. When I run the server and go to homepage also some others endpoint get result shown on the screenshot

<img width="1297" alt="Screenshot 2022-05-08 at 23 27 13" src="https://user-images.githubusercontent.com/99798843/167314978-1a5e971a-0d6a-4ea7-8e65-6350d5b8d9c5.png">
.
It seams that the folder **dist** is not seen by the server as it can not fin files specifically in this folder. It also seams that those files are expected to be in static root , but that did not work as well. 
Static files to the APIs that are using UI from rest framework or django admin (other folders in static folder) are working.
It did work smoothly with older Netbox version.

**Comment by DanSheps** [comment]
Thank you for opening a bug report. I was unable to reproduce the reported behavior on NetBox v3.2.2. Please re-confirm the reported behavior on the current stable release and adjust your post above as necessary. Remember to provide detailed steps that someone else can follow using a clean installation of NetBox to reproduce the issue. Remember to include the steps taken to create any initial objects or other data.

**Comment by jeremystretch** [comment]
Moving this to a discussion as the error indicates a configuration issue, not a bug.

**Comment by jeremystretch** [comment]
> 4. Fix and run migrations (as there is a migration issue with most migration files that were squashed)

Your first step should be to start over with the original migration files. There is no need to modify any of them. If you encounter an issue applying database migrations, you'll need to determine the cause before proceeding.
